### PR TITLE
Plot option for `get_current`

### PR DESCRIPTION
**Pull request by soerenjalas**
The LpaDiagnostic method `get_current()` now has a plotting flag and returns a FieldMetaInformation object with the central positions of the particle bins.
Additionally I added the possibility to pass keywords to matplotlib via `**kw` for all plotting methods in LpaDiagnostic.

### CODE REVIEW

File: opmd_viewer/addons/pic/lpa_diagnostics.py [hunk 1]
```diff
@@ -230,11 +230,16 @@ def get_current( self, t=None, iteration=None, species=None, select=None,
...
+           Whether to plot the requested quantity
+
+        **kw : dict, otional
+           Additional options to be passed to matplotlib's `plot` method
         Returns
         -------
         A tuple of arrays containig
         - The current in each bin in Ampere
-        - The z positions of the bin edges
+        - The z positions of the bin centers
```
Comment (RemiLehe) [inline opmd_viewer/addons/pic/lpa_diagnostics.py]: I would replace the line
```python
- The z positions of the bin centers
```
with
```python
- FieldMetaInformation object (see its docstring for more info)
```

**Comment by RemiLehe** [comment]
Thanks! I tested the features that you added in this PR, and it works great!
I had just a minor comment on the docstring, but once this is fixed, I'll merge the PR.

File: opmd_viewer/addons/pic/lpa_diagnostics.py (hunk 1 above)
Comment (soerenjalas) [inline opmd_viewer/addons/pic/lpa_diagnostics.py]: Oh I forgot to change the docstring…

**Comment by RemiLehe** [comment]
Thanks for the quick fix. This is now merged.

**Pull request merged**